src/client/app/components/Graph.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import styles from './styling/Graph.css';
+
+class Graph extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+
```

Comment:
Should this be stateless? Are you planning on adding methods that will set state on this component later?